### PR TITLE
fix(api_keys): improved the error message when bootstrapping api_key

### PR DESCRIPTION
**Pull request by lafirest**
Fixes EMQX-12210

Release version: v/e5.7.2

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [ ] Added tests for the changes
- [ ] Added property-based tests for code which performs user input validation
- [ ] Changed lines covered in coverage report
- [ ] Change log has been added to `changes/(ce|ee)/(feat|perf|fix|breaking)-<PR-id>.en.md` files
- [ ] For internal contributor: there is a jira ticket to track this change
- [ ] Created PR to [emqx-docs](https://github.com/emqx/emqx-docs) if documentation update is required, or link to a follow-up jira ticket
- [ ] Schema changes are backward compatible

## Checklist for CI (.github/workflows) changes

- [ ] If changed package build workflow, pass [this action](https://github.com/emqx/emqx/actions/workflows/build_packages.yaml) (manual trigger)
- [ ] Change log has been added to `changes/` dir for user-facing artifacts update

### CODE REVIEW

File: apps/emqx_management/src/emqx_mgmt_auth.erl [hunk 1]
```diff
@@ -365,6 +369,7 @@ init_bootstrap_file(File) ->
                     reason => Reason
```
Comment (zhongwencool) [inline apps/emqx_management/src/emqx_mgmt_auth.erl:369]: msg has been printed in the log. There should be no need to repeat it.
only return the format-reason but print explain-reason in log.

File: apps/emqx_management/src/emqx_mgmt_auth.erl [hunk 2]
```diff
@@ -356,7 +356,11 @@ init_bootstrap_file(File) ->
             {ok, MP} = re:compile(<<"(\.+):(\.+)(?::(\.+))?$">>, [ungreedy]),
             init_bootstrap_file(File, Dev, MP);
         {error, Reason0} ->
-            Reason = emqx_utils:explain_posix(Reason0),
+            Reason = io:format(
+                "load API bootstrap file failed, file:~ts, reason:~ts",
+                [File, emqx_utils:explain_posix(Reason0)]
+            ),
```
Comment (zhongwencool) [inline apps/emqx_management/src/emqx_mgmt_auth.erl:362]:
```suggestion
           Reason = emqx_utils:explain_posix(Reason0),
            FmtReason = emqx_utils:format((
                "load API-Key's bootstrap file failed, file:~ts, reason:~ts",
                [File, Reason]
            ),
```

**Review by zhongwencool — CHANGES_REQUESTED** [review]

File: apps/emqx_management/src/emqx_mgmt_auth.erl (hunk 1 above)
Comment (lafirest) [reply apps/emqx_management/src/emqx_mgmt_auth.erl:369]: No, most of the time this function is called at startup and will cause EMQX to crash, thus failing to print messages in the log file.

**Review by zhongwencool — APPROVED** [review]

**Comment by emqxqa** [comment]
[TestExecution](https://emqx.atlassian.net/browse/QA-2888)

**Pull request merged**